backend/cache/cache_internal_test.go

```diff
@@ -841,7 +841,7 @@ func newRun() *run {
 	}
 
 	if uploadDir == "" {
-		r.tmpUploadDir, err = ioutil.TempDir("", "rclonecache-tmp")
+		r.tmpUploadDir, err = ioutil.TempDir(testy.GetTempDir(), "rclonecache-tmp")
```

Comment:
I think this commit is bigger than it needs to be.

I think I would have just set the correct environment variable `TMPDIR` or `TMP` depending on OS. This means it will work also in libraries that we call, and will be inherited by any processes we fork, both of which are desirable.

Reply:
> I think I would have just set the correct environment variable TMPDIR or TMP (in the config routines) depending on OS.

Not very fond of this alternative. 😠 

> This means it will work also in libraries that we call, and will be inherited by any processes we fork, both of which are desirable.

But now you've convinced me! 😆